modules/core/src/main/resources/scalafix-migrations.conf

```diff
@@ -205,5 +205,11 @@ migrations = [
     newVersion: "0.14.0",
     rewriteRules: ["github:trace4cats/trace4cats-scalafix/v0_14?sha=v0.14.0"],
     doc: "https://github.com/trace4cats/trace4cats-docs/blob/master/docs/migrating.md"
-  }
+  },
+    {
+    groupId: "org.scala-lang.modules",
```

Comment:
**question (blocking):** Shouldn't this be run when updating Scala itself (instead of `scala-collection-compat`)?

<sub>This comment follows the [**conventionalcomments.org**](https://conventionalcomments.org) standard</sub>

Reply:
That was my remark: I have the feeling that the rule is too specific to the collection itself while it should be more oriented to the scala version. 
In this case, what should be the value of `newVersion` ? "2.13.0" or we case use a expression like 'newVersion = 2.13. ' to match any update to Scala 2.13 ?

Reply:
Yeah, I believe this should be:

```conf
  {
    groupId: "org.scala-lang",
    artifactIds: ["scala-library", "scala-compiler", "scala-reflect", "scalap"],
    newVersion: "2.13.0",
    doc: "https://github.com/scala/scala-collection-compat#collection213upgrade",
    rewriteRules: ["dependency:Collection213Upgrade@org.scala-lang.modules:scala-collection-migrations:2.8.1"]
  },
```

Reply:
WDYT @exoego?

Reply:
Yeah, I think this makes more sense.

The docs of that rewrite also mention that this Scalafix requires `scalacOptions: ["-P:semanticdb:synthetics:on"]`.

> In this case, what should be the value of `newVersion` ? "2.13.0" or we case use a expression like 'newVersion = 2.13. ' to match any update to Scala 2.13 ?

`2.13.0` is fine because Scala Steward will use that the rule when it updates from a version `< 2.13.0` to a version `>= 2.13.0`.

Reply:
Then, if we all agree I can update the PR with your feedbacks?

Reply:
@fthomas @alejandrohdezma thanks for the feedback, I've rebased and updated the PR. 
Tell me if it's good to you.